apps/rush-lib/src/schemas/deploy-scenario.schema.json

```
@@ -40,6 +40,11 @@
       "enum": ["default", "script", "none"]
     },
 
+    "postCopySourceFolder": {
+      "description": "If this path is specified, then after \"rush deploy\", recursively copies the files from this folder to the deployment target folder (common/deploy). This can be used to provide additional configuration files or scripts needed by the server when deploying.",
```

```suggestion
      "description": "If this path is specified, then after \"rush deploy\", recursively copy the files from this folder to the deployment target folder (common/deploy). This can be used to provide additional configuration files or scripts needed by the server when deploying.",
```

Also this copies into the deployment's subfolder, right?